pyximport/pyximport.py

```diff
@@ -355,7 +355,7 @@ def __init__(self, pyxbuild_dir=None, inplace=False, language_level=None):
     def find_module(self, fullname, package_path=None):
         if fullname in sys.modules:
             return None
-        if fullname.startswith('Cython.'):
+        if fullname.startswith('Cython.') or fullname.startswith('distutils.'):
```

Comment:
I'd rather have this depend on `blocked_modules`.

Reply:
Compiling .pyx sourcecode is failing only due `distutils.extension` module:

```
ImportError: Building module calc_pi failed: ["TypeError: pattern is not of type str nor subclass of Extension (<class 'distutils.extension.Extension'>) but of type <class 'distutils.extension.Extension'> and class <class 'distutils.extension.Extension'>\n"]
```

Do you want just to add this module to the `blcoked_modules`? Or you want to include there **all** distutils modules? (this can be pretty dirty since distutils contains **a lot** of modules)